example/wde/main.go

```diff
@@ -17,8 +17,7 @@ func events(events <-chan interface{}) {
 	for ei := range events {
 		switch e := ei.(type) {
 		case wde.MouseUpEvent:
-			switch e.Which {
-			case wde.LeftButton:
+			if e.Which == wde.LeftButton {
```

Comment:
I'd prefer to leave this as a switch -- even though it's pointless here, there's an advantage to having this example code mimic the usual event-loop code structure.

Reply:
Well, not according to gocritic linter ;)
Restoring the switch would require a `nolint` tag - acceptable?